qgis_deployment_toolbelt/jobs/job_qgis_installation_finder.py

```diff
@@ -45,19 +46,31 @@ class JobQgisInstallationFinder(GenericJob):
 
     ID: str = "qgis-installation-finder"
     OPTIONS_SCHEMA: dict = {
+        "if_not_found": {
+            "type": str,
+            "required": False,
+            "default": "warning",
+            "possible_values": ("warning", "error"),
+            "condition": "in",
+        },
         "version_priority": {
             "type": (list, str),
             "required": False,
             "default": None,
             "possible_values": None,
             "condition": None,
         },
-        "if_not_found": {
-            "type": str,
+        "search_paths": {
+            "type": (list, str),
             "required": False,
-            "default": "warning",
-            "possible_values": ("warning", "error"),
-            "condition": "in",
+            "default": (
+                expandvars("%PROGRAMFILES%"),
```

Comment:
I don't think it should be added because we're not looking at the subdirectories.

Reply:
See with @Guts I need to check subdirectories